governance/votes/Executive vote - October 04, 2024.md

```diff
@@ -0,0 +1,158 @@
+- Increase WBTC-C SF by 1.5 percentage points from 7.5% to **9%**.
+
+#### DSR and SSR Changes
+
+- Decrease DSR by 0.5 percentage points from 6% to **5.5%**.
+- Increase SSR by 0.25 percentage points from 6.25% to **6.5%**.
+
+The current Spark DAI market [Interest Rate Model (IRM)](https://sky-atlas.powerhouse.io/#A.3.8.1.5.1.3_Interest_Rate_Model_Definition-83e3f4f2-5e8e-4383-95e5-3af1bfb19210|57eaf45219be608847d6) is tied to the DSR. Consequently, the 50 basis point reduction in DSR will result in an approximate, but not exact, 50 basis point decrease in the Effective DAI borrow rate on SparkLend.
+
+### LITE-PSM-USDC-A Migration Phase 3
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183/2), [Poll 1143](https://vote.makerdao.com/polling/QmRjrFYG)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183)
+
+If this executive proposal passes, then the [third and final set of actions](https://forum.sky.money/t/lite-psm-usdc-a-phase-3-final-migration-proposed-parameters/25183) in the [migration process from USDC-PSM-A to LITE-PSM-USDC-A](https://forum.sky.money/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512) will be executed. This set of actions comprises the following:
+
+- Update PSM-USDC-A fees.
+- Migrate all remaining USDC reserves from PSM-USDC-A to LITE-PSM-USDC-A.
+- Update the PSM-USDC-A [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+- Increase the [LITE-PSM-USDC-A Buffer (`buf`)](https://sky-atlas.powerhouse.io/#A.3.4.1.5.1.1.6_Buffer_Definition-cf8e3d20-fffa-4b6a-9bf2-169d493c3638|57ea599773d45150b2632df9).
+- Update the LITE-PSM-USDC-A [DC-IAM](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+
+These actions are described below.
+
+#### PSM-USDC-A Fee Update
+
+- Decrease [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) by 0.01 percentage points from 0.01% to **0%**.
+- Decrease [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) by 0.01 percentage points from 0.01% to **0%**.
+
+#### Remaining USDC Migration
+
+The remaining USDC in [PSM-USDC-A](https://makerburn.com/#/collateral/PSM-USDC-A) will be migrated to [LITE-PSM-USDC-A](https://makerburn.com/#/collateral/LITE-PSM-USDC-A) through a script executed in the spell.
+
+#### PSM-USDC-A DC-IAM Update
+
+- Decrease the [Maximum Debt Ceiling (`line`)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4.1_Maximum_Debt_Ceiling_(line)-6f1a913d-9436-4b70-816b-e317672737d6|57eaf45219bea3b430c268bb) by 2,5 billion DAI from 2,5 billion DAI to **0 DAI**.
+- Disable the PSM-USDC-A DC-IAM.
```

Comment:
This is technically incorrect and does not reflect what will happen in the code. The `line` cannot be set to 0 - [see code](https://github.com/makerdao/dss-auto-line/blob/master/src/DssAutoLine.sol#L83).

We suggest:

- Remove PSM-USDC-A from the Debt Ceiling Instant Access Module (DC-IAM)
- Set PSM-USDC-A Debt Ceiling to **0 DAI**

This matches [previous notation](https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20August%2012%2C%202024.md?plain=1#L48)